validate.lic

```diff
@@ -678,6 +678,18 @@ class DRYamlValidator
             .any? { |skill| error("Skill name in cycle_armors_regalia is not valid: #{skill}") }
   end
 
+  def assert_that_non_training_spells_are_in_base_spells(settings)
+    return unless settings.offensive_spells || settings.buff_spells
+
+    spell_names = []
+    settings.buff_spells.each { |spell| spell_names.push(spell.first) }
+    settings.offensive_spells.each { |spell| spell_names.push(spell['name']) }
+    spell_data = get_data('spells').spell_data
+
+    spell_names.reject { |name| spell_data.keys.include?(name) }
+               .each { |name| warn("Spell not found in base_spells '#{name}'! Check its spelling and capitalization.") }
```

Comment:
💡 What do you think about also outputting in the warning the name of the setting that needs fixed? e.g. `buff_spells` or `offensive_spells`

Reply:
Sure, more info more better.